sccm/apps/deploy-use/import-export-applications.md

```diff
@@ -0,0 +1,46 @@
+---
+
```

Comment:
Here's how the metadata block should look, starting at line 1 of the article:

```
---
title: Import and export applications
titleSuffix: Configuration Manager
description: Learn how to import and export applications in Configuration Manager to share between separate hierarchies.
ms.date: 05/23/2019
ms.prod: configuration-manager
ms.technology: configmgr-app
ms.topic: conceptual
ms.assetid: dba00e54-9d5b-4f6b-916d-ead48c66e288
author: aczechowski
ms.author: aaroncz
manager: dougeby
ms.collection: M365-identity-device-management
---
```

- I'm hoping if you set the author field to me, the bot will let you reopen this PR 
- The asset ID is just a random GUID that I generated via New-Guid cmdlet 😉

Reply:
Fixed in #1585